src/main/java/org/ow2/proactive/catalogclient/service/CatalogObjectService.java

```diff
@@ -118,19 +118,19 @@ public String getResolvedCatalogObject(String catalogUrl, long bucketId, String
     /**
      * Generate the catalog URL from the argument
      * @param catalogURL is the domain name
-     * @param bucketId is the id of bucket containing the resource
+     * @param bucketName is the id of bucket containing the resource
      * @param name is the resource name
      * @param raw enables to choose for the resource metadata or the resource content
      * @return the resource content if raw is true otherwise return the resource metadata
      */
     @VisibleForTesting
-    String getURL(String catalogURL, long bucketId, String name, boolean raw) {
+    String getURL(String catalogURL, String bucketName, String name, boolean raw) {
 
         final String bucketsPath = "buckets/";
         final String resourcesPath = "/resources/";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)